pkgs/applications/misc/archivy/default.nix

```diff
@@ -49,21 +50,9 @@ buildPythonApplication rec {
     hash = "sha256-o5dVJDbdKgo6hMMU9mKzoouSgVWl7xSAp+Aq61VcfeU=";
   };
 
-  # Relax some dependencies
-  postPatch = ''
-    substituteInPlace requirements.txt \
-      --replace 'WTForms ==' 'WTForms >=' \
-      --replace 'attrs == 20.2.0' 'attrs' \
-      --replace 'elasticsearch ==' 'elasticsearch >=' \
-      --replace 'python_dotenv ==' 'python_dotenv >=' \
-      --replace 'python_frontmatter == 0.5.0' 'python_frontmatter' \
-      --replace 'requests ==' 'requests >=' \
-      --replace 'validators ==' 'validators >=' \
-      --replace 'flask-login == ' 'flask-login >= ' \
-      --replace 'tinydb ==' 'tinydb >=' \
-      --replace 'Flask_WTF == 0.14.3' 'Flask_WTF' \
-      --replace 'Flask ==' 'Flask >='
-  '';
+  nativeBuildInputs = [ wheelDepsVersionsHook ];
+
+  relaxPythonDeps = true;
```

Comment:
Note that for relaxing all dependencies it is also possible to write
```
pipInstallFlags = [ "--ignore-requires-python" ];
```

Reply:
If I understand this flag correctly it would completely ignore the `Requires-Dist` right? It means them that if the user forgets to add a dependency to `buildInputs` this will not make the build fails anymore, but will result in runtime failures right?

In this case, I think `relaxPythonDeps = true;` is still a better option since it will only remove the version restrictions.

Reply:
Ah sure 😅 .

Yeah, I still don't think if I found an actual user case for `removePythonDeps = true;`. I mostly implemented it to keep the interface consistency.